src/JwksClient.js

```diff
@@ -36,9 +42,21 @@ export class JwksClient {
   }
 
   getKeys(cb) {
-    this.logger(`Fetching keys from '${this.options.jwksUri}'`);
+    if (Array.isArray(this.options.jwks)) {
```

Comment:
I would expect a jwks to be an object with a `keys` property https://tools.ietf.org/html/rfc7517#page-25